app/src/androidMain/kotlin/de/westnordost/streetcomplete/data/quest/atp/CreatePoiBasedOnAtp.kt

```diff
@@ -0,0 +1,22 @@
+package de.westnordost.streetcomplete.data.quest.atp
+
+import de.westnordost.osmfeatures.Feature
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.mapdata.Element
+import de.westnordost.streetcomplete.data.osm.mapdata.MapDataWithGeometry
+import de.westnordost.streetcomplete.data.quest.AndroidQuest
+import de.westnordost.streetcomplete.data.quest.OsmCreateElementQuestType
+import de.westnordost.streetcomplete.data.user.achievements.EditTypeAchievement.CITIZEN
+import de.westnordost.streetcomplete.osm.isPlaceOrDisusedPlace
+
+class CreatePoiBasedOnAtp() : OsmCreateElementQuestType<CreatePoiBasedOnAtpAnswer>, AndroidQuest {
```

Comment:
```suggestion
object CreatePoiBasedOnAtp : OsmCreateElementQuestType<CreatePoiBasedOnAtpAnswer>, AndroidQuest {
```

Make this an `object`, just like `OsmNoteQuestType`